compute/client_library/snippets/tests/test_disks.py

```diff
@@ -173,3 +245,26 @@ def test_create_and_delete_regional_disk(test_snapshot):
         assert regional_disk.type_.endswith(disk_type)
     finally:
         delete_regional_disk(PROJECT, REGION, disk_name)
+
+
+def test_disk_attachment(autodelete_blank_disk, autodelete_regional_blank_disk, autodelete_compute_instance):
+    instance = get_instance(PROJECT, ZONE, autodelete_compute_instance.name)
+
+    assert len(list(instance.disks)) == 1
+
+    attach_disk(PROJECT, ZONE, instance.name, autodelete_blank_disk.self_link)
+    attach_disk(PROJECT, ZONE, instance.name, autodelete_regional_blank_disk.self_link)
```

Comment:
Does it make sense to test read_write mode as well?

Reply:
read_write mode is still in preview: https://cloud.google.com/compute/docs/disks/sharing-disks-between-vms#multi_writer_mode

Reply:
Interesting, so will the sample work if we pass READ_WRITE in mode parameter? If the library doesn't support it yet then probably we should not include this in the comments. If we do support it then I think we should add a test.